cmd/root.go

```diff
@@ -100,16 +100,71 @@ func WithDialer(d cloudsql.Dialer) Option {
 	}
 }
 
+var longHelp = `
+The Cloud SQL Auth proxy is a utility for ensuring secure connections to your
+Cloud SQL instances. It provides IAM authorization, allowing you to control who
```

Comment:
The proxy handles authorization to the database using Google Cloud IAM permissions. The proxy configures TLS 1.3 encryption for connections to the database so that your application does not need to manage certificates.

Reply:
Getting the wording right on this one is key. I think we start with a high level sentence (proxy as a utility for secure connections), and then expand with details (IAM auth, TLS 1.3, etc.). Because there's a user authentication step, people easily confuse the proxy's authorization with logging in to the database. I'm taking the wording from here: https://github.com/GoogleCloudPlatform/cloudsql-proxy/pull/1273/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R5.